### PR TITLE
Refactor repository to make pip installable gear package.

### PR DESCRIPTION
**Pull request by margaretmahan**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"""
+    dcm2niix-gear
+"""
+import setuptools
+
+VERSION = "1.2.2_1.0.20201102"
```
Comment (NPann) [inline setup.py:6]: What about parsing it from the manifest.json directly to avoid human error?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+with open("README.md", "r") as f:
+    LONG_DESCRIPTION = f.read()
+
+setuptools.setup(
+    name="dcm2niix-gear",
```
Comment (NPann) [inline setup.py:12]: Would rename as "fw-gear-dcm2niix". In gitlab that's how we have started enforcing this. The idea being to group all gear related packages under a common prefix.

**Review by NPann — CHANGES_REQUESTED** [review]
LGTM. Just the few minors comments below in prep for the migration to gitlab.

**Review by NPann — APPROVED** [review]

**Review by qian587 — APPROVED** [review]
Looks good to me too!

**Pull request merged**